ComfortZoneApp/app/src/main/java/com/example/comfortzone/input/ui/InputFragment.java

```diff
@@ -100,6 +104,14 @@ private void getWeatherClass() {
         LocationUtil.getLastLocation(getActivity(), new LocationCallback() {
             @Override
             public void onLocationUpdated(String lat, String lon) {
+                IataClient iataClient = new IataClient();
+                iataClient.getIataResponse(lat, lon, new IataCallback() {
+                    @Override
+                    public void onGetIata(String iata) {
+                        LOC_IATA = iata;
```

Comment:
hmm, I would avoid using static variable to store mutable values, plus static values live the entire lifecycle of the app, so we can't recollect its memory if we need to. Maybe it will be easier to manage this once you move these to HostActivity